### PR TITLE
Improved behavior of the run shortcut key (ctrl+return (ctrl+enter)) on Apple devices

### PR DESCRIPTION
**Pull request by liberaldev**
At the time I posted #183, I didn't have a Mac or other Apple device, so I couldn't validate the issue with Apple devices. I have now purchased a Mac and verified the issue that occurs on Apple devices, and this PR improves the run shortcut to work well on Apple devices. The detailed improvements are as follows:

- Added cmd + return shortcut
- Improved conflict in Safari by adding e.prevent.
- iOS and iPadOS that still conflict now only work with cmd + return.

### CODE REVIEW

File: app/try_ruby.rb [hunk 1]
```diff
@@ -61,7 +62,9 @@ def initialize
 
     #If hold down the control and the Enter key goes down, run
     $document.on :keydown, '#editor' do |e|
-      if e.key == "Enter" && e.ctrl?
+      if e.key == "Enter" && ((e.ctrl? && !@navigator.user_agent&.match?(/\b(iPad|iPhone|iPod)\b/)) ||
```
Comment (hmdne) [inline app/try_ruby.rb]: This is kinda hard to read... maybe we could add helper functions like:

```ruby
def ios?
  @navigator.user_agent&.match?(/\b(iPad|iPhone|iPod)\b/)
end

def macos?
  @navigator.user_agent&.match?(/\bMac\b/)
end
```

Then the modified code could be just:

```ruby
    $document.on :keydown, '#editor' do |e|
      if e.key == "Enter" && ((!ios? && e.ctrl?) || ((ios? || macos?) && e.meta?))
        e.prevent
        do_run
      end
    end
```

Comment (liberaldev) [reply app/try_ruby.rb]: I added the Helper class to make my code more reusable

File: app/try_ruby.rb [hunk 2]
```diff
@@ -61,7 +62,8 @@ def initialize
 
     #If hold down the control and the Enter key goes down, run
     $document.on :keydown, '#editor' do |e|
-      if e.key == "Enter" && e.ctrl?
+      if (e.key == "Enter" && (e.ctrl? && !Helper.ios?)) || ((Helper.macos? || Helper.ios?) && e.meta?)
```
Comment (hmdne) [inline app/try_ruby.rb]: Right-hand side of the || expression is missing check for `Enter`.

Comment (liberaldev) [reply app/try_ruby.rb]: Thank you for pointing this out. I've fixed the issue, please check it out.

**Comment by hmdne** [comment]
Looks good to me :D Let's merge

**Pull request merged**